cmd/svcat/extra/marketplace_cmd.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package extra
```

Comment:
What do you think about putting this under `cmd/svcat/plan` which has other verbs related to displaying and interacting with plans?

Reply:
so, I considered about where to put this - it's only sort of related to only classes or only plans. I figured we might as well have a place for commands that aren't strictly related to one of the types, and I didn't want to name it `commands` because we already have a `command` dir in there. I was planning on moving the other commands like completion and install here at some point. I'm open to suggestions for a better name.